### PR TITLE
Update chineseSimplified.xml

### PR DESCRIPTION
**Pull request by popcorner**
translate new strings in english.xml

### CODE REVIEW

File: PowerEditor/installer/nativeLang/chineseSimplified.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
                     <Item menuId="encoding" name="格式(&amp;M)" />
                     <Item menuId="language" name="语言(&amp;L)" />
                     <Item menuId="settings" name="设置(&amp;T)" />
+                    <Item menuId="tools" name="工具(&amp;O)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/chineseSimplified.xml]: Problem: there are 2 `&amp O`
Could you please aligned to chinese.xml, please?
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/chinese.xml

**Comment by popcorner** [comment]
@donho problem solved
And fix a wrong translation at the same time

**Comment by donho** [comment]
@popcorner Sorry again for the request, but I just add a new feature and several new translations
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/6a05bedd05cbfce40106aa464e43448639d1913b#diff-2a8496e2537c34b95cbeca007cd2346b

Could you update the PR again please? (so I can include it in new release)

**Comment by popcorner** [comment]
@donho Done

**Comment by donho** [comment]
@popcorner Thank you!

**Pull request closed without merging**